File: server/makc2022--nodejs--nestjs/src/layer4/domains/dummy-main/queries/list/get/domain-list-get-query-output.ts
/** Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License. */

import { ListGetQueryOutput } from 'src/layer2/nosql-mongo/queries/list/get/item-list-query-output';
import { DomainItemGetQueryOutput } from '../../item/get/domain-item-get-query-output';

/** Выходные данные запроса на получение списка в домене. */
export class DomainListGetQueryOutput extends ListGetQueryOutput<DomainItemGetQueryOutput> {}

File: server/makc2022--dotnet/Makc2022.Layer4.Sql.Domains.DummyMain/Queries/List/Get/DomainListGetQueryOutput.cs
// Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License.

using Makc2022.Layer2.Sql.Queries.List.Get;
using Makc2022.Layer4.Sql.Domains.DummyMain.Queries.Item.Get;

namespace Makc2022.Layer4.Sql.Domains.DummyMain.Queries.List.Get
{
    /// <summary>
    /// Выходные данные запроса на получение списка в домене.
    /// </summary>
    public class DomainListGetQueryOutput : ListGetQueryOutput<DomainItemGetQueryOutput>
    {
    }
}
